### ISSUE TITLE
Batch Normalization

### ISSUE THREAD

**Issue by 425her0** [bubbliiiing/unet-pytorch#42]
博主，我想问一下我在网络中增加了batchnorm，它的均值和方差是按照一个batch来计算的。但是我看在计算miou时，图片是一张一张做预测的，这个会有影响吗？

**Comment by bubbliiiing** [comment]
不会
